base/friendly/friendly.go

```diff
@@ -107,38 +115,48 @@ func DiffDescriptions(deltas []*deepdiff.Delta, stats *deepdiff.Stats) (string,
 	return shortTitle, longMessage
 }
 
-// preprocess makes delta lists easier to work with, by combining operations when possible
-func preprocess(deltas []*deepdiff.Delta) []*deepdiff.Delta {
+const dtReplace = deepdiff.Operation("replace")
+
+
+
+// preprocess makes delta lists easier to work with, by combining operations 
+// when possible & removing unwanted paths
+func preprocess(deltas deepdiff.Deltas, path string) deepdiff.Deltas {
 	build := make([]*deepdiff.Delta, 0, len(deltas))
 	for i, d := range deltas {
 		if i > 0 {
 			last := build[len(build)-1]
-			if last.Path == d.Path {
+			if last.Path.String() == d.Path.String() {
 				if last.Type == deepdiff.DTDelete && d.Type == deepdiff.DTInsert {
-					last.Type = "replace"
+					last.Type = dtReplace
 					continue
 				}
 			}
 		}
+
+		p := joinPath(path, d.Path.String())
+		// TODO (b5) - We need this because it's possible to write a transform script
+		// that changes nothing other than the script itself, and we currently reject
+		// that change. Should we? I think I'm missing something.
```

Comment:
Discussed offline, we should definitely *not* be rejecting a change that consists of a transform modification that produces no body diff. There are very good use cases for that sort of thing.